requirements/base.in

```diff
@@ -34,6 +34,7 @@ django-yubin
 mozilla-django-oidc-db
 -e git+https://bitbucket.org/maykinmedia/django-digid-eherkenning.git@0cb1978e7d628e7923dbd30ff5dbae6dbcc87129#egg=digid_eherkenning
 -e git+https://github.com/maykinmedia/python3-saml.git@58fd3ad798510c7ced110040d2fd0e6ad40d02da#egg=python3_saml
+-e git+https://github.com/maykinmedia/mozilla-django-oidc-db.git@806ba7a0448ecaae0c1271b4afa2f282601e8a6a#egg=mozilla-django-oidc-db
```

Comment:
```suggestion
# replace with new PyPI version once https://github.com/maykinmedia/mozilla-django-oidc-db/pull/14 is merged and released
git+https://github.com/maykinmedia/mozilla-django-oidc-db.git@806ba7a0448ecaae0c1271b4afa2f282601e8a6a#egg=mozilla-django-oidc-db
```

This also solves the problem of the package being put in the `src` directory and breaking the isort checks